test/runnable/test42.d

```diff
@@ -1,5 +1,20 @@
 // REQUIRED_ARGS:
-//
+/*
+TEST_OUTPUT:
+---
+success
+Not ported to x86-64 compatible varargs, yet.
+../../phobos/std/format.d(1854): Deprecation: function `std.format.formatValueImpl!(LockingTextWriter, ifloat, char).formatValueImpl` is deprecated - Use of imaginary types is deprecated. Use std.complex
```

Comment:
Should remove this because of phobos dependency for now.

Also... output differs depending on platform.